_includes/markdown/Markup.md

```diff
@@ -233,6 +233,26 @@ Form elements should also be logically grouped using the ```<fieldset>``` elemen
 
 Finally, we should ensure that all interactive elements are keyboard navigable, providing easy use for people with vision or mobility disabilities (or those not able to use a mouse). In general, the tab order should be dictated by a logical source order of elements. If you feel the need to change the tab order of certain elements, it likely indicates that you should rework the markup to flow in a logical order or have a conversation with the designer about updates that can be made to the layout to improve accessibility.
 
+### Bypass Blocks
+
+[Bypass blocks](https://developer.wordpress.org/themes/functionality/accessibility/#skip-links), sometimes knows as 'skip to content links' - or simply 'skip links' - are links in a document to allow users that rely on screen readers, keyboard navigation or other assistive technologies to _bypass_ certain page elements or _skip_ to a specific section of a page with ease.
```

Comment:
@pattonwebz Bypass are more than just skip links, it's also aria-roles and some other method of chunking parts of a document for assistive technology to use. Here's some more information: https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#bypass-blocks